Documentation/api.md

```diff
@@ -745,6 +745,7 @@ RemoteWriteSpec defines the remote_write configuration for prometheus.
 | ----- | ----------- | ------ | -------- |
 | url | The URL of the endpoint to send samples to. | string | true |
 | name | The name of the remote write queue, must be unique if specified. The name is used in metrics and logging in order to differentiate queues. Only valid in Prometheus versions 2.15.0 and newer. | string | false |
+| sendExemplars | Enables sending of exemplars over remote write. Note that exemplar storage itself must be enabled for exemplars to be scraped in the first place.  Only valid in Prometheus versions 2.27.0 and newer. | boolean | false |
```

Comment:
This file is generated from `pkg/apis/monitoring/v1/types.go` so the comment needs to go there.

Reply:
Maybe also reference the `enableFeatures` field.